spec/mailers/feedback_mailer_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FeedbackMailer, type: :mailer do
+  let(:feedback_template) { '5fcd84b1-8d05-4b6c-86a8-9fc391efa754' }
+  let(:comment) { 'A feedback comment' }
+  let(:email) { 'feedback.user@example.com' }
+  let(:rating) { 1 }
+  let(:environment) { 'test' }
+
+  describe 'Feedback Email' do
```

Comment:
as a general rule unit tests are easier to understand when they follow the convention of `describe`ing a method and their description follows the convention of using `#` for instance methods and `.` for class methods - e.g. `#instance_methods`, `.class_method` 

```
# spec an instance method
describe '#notify` do
end
```

```
# spec a class method
describe '.my_class_method` do
end
```

Reply:
the `subject` is then a call to that method, as you have applied here

Reply:
have pushed a fix